src/tests/routes/controllers/articles.test.js

```diff
@@ -171,61 +191,74 @@ describe('Articles', () => {
-        .set('x-access-token', xAccessToken)
-        .send(articleRequestObject)
+        .delete(`/api/v1/articles/cc75f9de-324e-4b7e-be68-64c0ce09bc4d`)
+        .set('Authorization', xAccessToken)
         .end((err, res) => {
-          expect(res.status).to.eql(201);
-          expect(res.body.status).to.eql('success');
-          expect(res.body.message).to.eql(
-            'your article has been created successfully',
+          expect(res.status).to.eql(200);
+          expect(res.body.status).to.equal('success');
+          expect(res.body.message).to.equal(
+            'article as been deleted successfully',
           );
-          articleId = res.body.data.id;
           done();
         });
     });
-
-    it('should fail when id is invalid', done => {
+    it('should return 404 responses when article does not exist', done => {
       chai
         .request(app)
-        .delete(`/api/v1/articles/${articleId}ss`)
-        .set('Authorization', xAccessToken)
-        .send({})
+        .put(`${endPoint}/cc75f9de-324e-4b7e-be68-64c0ce09bc4d`)
+        .set('x-access-token', jwtToken)
+        .send(articleRequestObject)
         .end((err, res) => {
           expect(res.status).to.equal(404);
           expect(res.body.message).to.equal('article does not exist');
           done();
         });
     });
+  });
 
-    it('should fail when userid does not match articles users id', done => {
+  describe('UPDATE ARTICLE', () => {
+    const articleSlug = slug('why i kill the bird - fde6799275c1');
+    const articleSlugOther = slug('while i wait for you - 987say6tds3h');
+    it('should return 422 responses when there is validation error', done => {
       chai
         .request(app)
-        .delete(`/api/v1/articles/${articleId}`)
-        .set('Authorization', falseToken)
+        .put(`${endPoint}/${articleSlug}`)
+        .set('x-access-token', jwtToken)
         .send({})
         .end((err, res) => {
-          expect(res.status).to.equal(401);
-          expect(res.body.message).to.equal(
-            'you are not authorized to perform this action',
-          );
+          expect(res.status).to.equal(422);
+          expect(res.body.message).to.equal('validation error');
           done();
         });
     });
 
-    it('should delete article successfully, with valid user input', done => {
+    it('should return 200 responses when article is updated', done => {
       chai
         .request(app)
-        .delete(`/api/v1/articles/${articleId}`)
-        .set('Authorization', xAccessToken)
+        .put(`${endPoint}/${articleSlugOther}`)
+        .set('x-access-token', jwtToken)
+        .send(articleRequestObject)
         .end((err, res) => {
-          expect(res.status).to.eql(200);
-          expect(res.body.status).to.equal('success');
+          expect(res.status).to.equal(200);
+          expect(res.body.message).to.equal('article update successful');
+          expect(res.body.data).to.be.an('object');
```

Comment:
You forgot to test that the response from the server matches the article content you sent to it

Reply:
I will handle that, thanks